src/effect/fadeInDown.js

```diff
@@ -14,16 +14,14 @@ export default {
     'panel': {
         'position': 'fixed',
         'backgroundColor': '#fff',
-        'width': '400px',
-        'height': '300px',
+        'width': 'auto',
+        'height': 'auto',
         'top': '50%',
         'left': '50%',
-        'marginTop': '-150px',
-        'marginLeft': '-200px',
         'borderRadius': '5px',
         'boxSizing': 'border-box',
         'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.3)',
-        'transform': 'translate3d(0, 0, 0)',
+        'transform': 'translate3d(-50%, -50%, 0)',
```

Comment:
is this merged to master? I started to had a styles obj has config for the modal =) before I saw this one. I can do a pull request if it helps share the basic idea.